5.x supportin7.x

AnkurChauhan-131 opened the pull request:
Add Activiti 5 compatibility code with Activiti5 Engine compatible code

Commit:
Activiti 5 Support

Commit:
Revert Version changes

Commit:
delete html file

Commit:
Revert invalid revert changes

Commit:
Revert test case

Commit:
Refactor code

Commit:
Refactor code

balsarori reviewed (changes requested):
@AnkurChauhan-131 can you remove the org.activiti.form package and all classes in it? That is not required.

Commit:
Delete unwanted files

AnkurChauhan-131 commented:
> @AnkurChauhan-131 can you remove the org.activiti.form package and all classes in it? That is not required.

Deleted engine/form package. But it seems Activit5 code that we have added in APS do refer to the files within engine/form package. So should i add them in APS itself

balsarori commented:
@AnkurChauhan-131 they should be deleted from there too

balsarori commented on activiti-core/activiti-engine/src/main/java/org/activiti/engine/FormService.java:
```diff
@@ -0,0 +1,113 @@
+/*
```
This file can be removed

Commit:
Remove unwanted files

Commit:
Remove unwanted method

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Activiti/Activiti?pullRequest=3738) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Activiti/Activiti?pullRequest=3738) before we can accept your contribution.<br/><hr/>**Your Name** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Activiti/Activiti?pullRequest=3738) it.</sub>

erdemedeiros commented:
Covered by https://github.com/Activiti/Activiti/pull/3740
